jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.vue

```diff
@@ -50,14 +50,14 @@
               <v-text-field
               v-model="filename"
               label="Filename"
-              hint="Export the latest extracted spectrum"
+              hint="Export the latest extracted spectrum."
               :rules="[() => !!filename || 'This field is required']"
               persistent-hint>
               </v-text-field>
           </v-row>
 
           <v-row justify="end">
-            <j-tooltip tipid='plugin-collapse-save-fits'>
+            <j-tooltip tipid='plugin-extract-save-fits'>
```

Comment:
these need corresponding entries in `tooltip.vue`

Reply:
done